media-gfx/wings/wings-2.1.7-r1.ebuild

```diff
@@ -13,7 +13,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 RDEPEND="
-	>=dev-lang/erlang-18.1[smp,wxwidgets]
+	>=dev-lang/erlang-18.1[wxwidgets]
```

Comment:
To ensure compatibility with previous erlang ebuild, wouldn't it be better to use `smp(+)`? See https://devmanual.gentoo.org/general-concepts/dependencies/

Reply:
Fixed.